pom.xml

```diff
@@ -265,7 +265,6 @@
             <version>${system-rules.version}</version>
             <scope>test</scope>
         </dependency>
-
```

Comment:
don't understand why this changed, but I'll ignore it this time
just be careful next time

On bigger PRs all those little changes are annoying
It also messes up hisotry